mgmt/rest/v1/plugin.go

```diff
@@ -176,6 +177,27 @@ func (s *apiV1) loadPlugin(w http.ResponseWriter, r *http.Request, _ httprouter.
 		}
 		lp.LoadedPlugins = append(lp.LoadedPlugins, catalogedPluginToLoaded(r.Host, pl))
 		rbody.Write(201, lp, w)
+	} else if strings.HasSuffix(mediaType, "json") {
+		body, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			rbody.Write(500, rbody.FromError(err), w)
+			return
+		}
+		var resp map[string]string
+		err = json.Unmarshal(body, &resp)
```

Comment:
The error should be handled here. It should be logged as log.Fatal(err) or log.Fatal("invalid config")